text/0005-coretime-interface.md

```diff
@@ -0,0 +1,176 @@
+# RFC-5: Coretime Interface
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 06 July 2023                                                                                |
+| **Description** | Interface for manipulating the usage of cores on the Polkadot Ubiquitous Computer. |
+| **Authors**     | Gavin Wood                                                                                  |
```

Comment:
We spent quite a lot of time on `fn assign_core` and `request` core parts of things together - formal request for an author credit here